micro/service.go

```diff
@@ -35,7 +35,7 @@ type (
 	// Service exposes methods to operate on a service instance.
 	Service interface {
 		// AddEndpoint registers endpoint with given name on a specific subject.
-		AddEndpoint(name, subject string, handler Handler) error
+		AddEndpoint(string, Handler, ...EndpointOpt) error
```

Comment:
very small nit - should it be `EndpointOpts`? (yes we have only one, but otherwise, it should be OptSubject).

Reply:
No, the signature has a variadic arg and therefore represents 0 or more instances of `EndpointOpt`. Similarly you would have `[]EndpointOpt` if it's a slice.